app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/ChatRoom.java

```diff
@@ -26,6 +26,10 @@ public ChatRoom(String id) {
     }
 
     public void setMessages(List<Message> messages) {
+        if (messages == null) {
+            throw new IllegalArgumentException("Messages cannot be null");
```

Comment:
maybe change to a message cannot be blank since null message doesn't really make sense?

Reply:
This checks the list of messages not individual messages